### PR TITLE
chore: Generate statik.go files

### PR DESCRIPTION
**Pull request by manno**
Since we keep gettings conflicts for those.

refs #270

### CODE REVIEW

**Comment by manno** [comment]
Had to update and removed the 'embed_files' from test-acceptance target, since it's already called from acceptance_suite.go

File: .gitignore [hunk 1]
```diff
@@ -16,3 +16,6 @@ personal-setup
 /resources
 acceptance/acceptance.test
 .vscode
+statik/statik.go
```
Comment (andreas-kupries) [inline .gitignore:19]: Why a file here, versus directories below ?

**Review by andreas-kupries — CHANGES_REQUESTED** [review]
Just a question/nit.

File: .gitignore (hunk 1 above)
Comment (manno) [reply .gitignore]: 👍

**Comment by manno** [comment]
Talked with @jimmykarily and changed the getstatik to get a specific version of statik.

**Review by andreas-kupries — APPROVED** [review]

**Pull request merged**